src/components/poll/SubmitChoices.tsx

```diff
@@ -31,6 +44,16 @@ const SubmitChoices = (props: {
         };
         const submitChoiceResponse = await markChoices(voterArgs);
         if (submitChoiceResponse.statusCode === 201) {
+          if (isLoggedIn) {
+            const mailerArgs: MailerArgs = {
+              pollID,
+              senderEmailID,
+              pollTitle: pollFromDB.title,
+              receiverIDs: Array<string>(pollCreatorEmailID),
+              senderName: newVote.name,
+            };
+            sendPollResponse(mailerArgs, token);
```

Comment:
Just noticed, `await` is missing here.

Reply:
I did it deliberately in order to reduce waiting time. I thought as soon as the request is initiated, It is the job of mailer service to send mail which is independent of client which is working fine on local. Am I wrong in this logic @AnandBaburajan ?

Reply:
In production the request gets cancelled, so I looked up [this](https://stackoverflow.com/questions/12009423/what-does-status-canceled-for-a-resource-mean-in-chrome-developer-tools) SO answer which says, "[one] of the only places where Chrome will cancel a request [is when] the DOM element that caused the request to be made got deleted". So I think we need to use `await` and the current approach works locally because the request is fast and sent before the page is reloaded. Please correct me if I'm wrong somewhere! Thanks.

Reply:
I guess you are right. I need to add `await` keyword in the call. So I need to create a new PR for the changes or is it possible to reopen this one ?

Reply:
You can open a new PR.